layouts/_default/rss.xml

```diff
@@ -3,7 +3,7 @@
 {{- $pages := $pctx.RegularPages -}}
 {{- $limit := .Site.Config.Services.RSS.Limit -}}
 {{- if ge $limit 1 -}}
-{{- $pages = $pages | first $limit -}}
+{{- $pages = where $pages ".Params.disable_feed" "!=" true | first $limit -}}
```

Comment:
I think this condition should be on 3rd line, because this line is applied only for case with rss limit. 
Also, maybe it's better to name parameter "disable_rss" for better clarification?

Reply:
The reason I use `disable_feed` is that it is also the variable used by https://github.com/kaushalmodi/hugo-atom-feed/ and would rather avoid having two different ways of saying the same thing.

Reply:
I moved it to the `{{- $pages := $pctx.RegularPages -}}` line, this ensures that the limit to the number of posts displayed in the feed is applied *after* filtering out posts that should not be displayed.